### DISCUSSION TITLE
Support Rainbow library for integrating frontend?

### DISCUSSION THREAD

**Discussion by personalcodecare** [WalletConnect/walletconnect-monorepo#5553] [Developer Support] [answered]
I need to integrating frontend web3 using rainbow library.
Is it possible to use it?

**Comment by rtomas** [comment, marked as answer]
Hi @personalcodecare ! I hope you are doing good. Can you give us more context ?

AppKit supports rainbow Wallet.
Are you referring Rainbow library to rainbow kit ?

Did you see our examples and also what you can build with AppKit ?
https://github.com/reown-com/appkit-web-examples/
